docs/01-Using-Fleet/FAQ.md

```diff
@@ -97,6 +97,13 @@ Expecting results, but not seeing anything in the logs?
 - Use live query to `SELECT * FROM osquery_schedule` to check whether the query has been scheduled on the host.
 - Look at the status logs provided by osquery. In a standard configuration these are available on the filesystem of the Fleet server at the path configurable by [`--filesystem_status_log_file`](../02-Deploying/02-Configuration.md#filesystem_status_log_file). This defaults to `/tmp/osquery_status`. The host will output a status log each time it executes the query.
 
+## Why does the same query come back faster sometimes?
+
+Don't worry, this behavior is expected; it's part of how osquery works.
+
+Fleet and osquery work together by communicating with heartbeats, much like a new mother and her newborn. Depending on how close the next heartbeat is, Fleet might return results a few seconds faster or slower.
```

Comment:
```suggestion
Fleet and osquery work together by communicating with heartbeats, much like a mother and her unborn baby. Depending on how close the next heartbeat is, Fleet might return results a few seconds faster or slower.
```

Reply:
I'm not a doctor, so I could be wrong 😅 but is it mothers and unborn babies that communicate through heartbeats while in the womb? Actually, I have no idea... I guess newborns are maybe comforted by heartbeats when cuddling mum?